docker-compose-dev.yaml

```diff
@@ -0,0 +1,32 @@
+### INSTRUCTIONS ###
+## The purpose of this docker container is to simplify bringing up Gaps while developing.
+# 1. Build Gaps Jar
+#     - mvn clean install
```

Comment:
This should change to `./minify && mvn clean install`

The minify script takes all the JS and CSS and runs them through eslint for checkstyle and minifies them.

Reply:
I would beg to differ here. The purpose of this dockerfile/docker-compose file is to ease applying development changes. The intent is not to build a production ready image. As such you typically don't want to minify JS until you're confident of the implementation. Makes it easier to debug/develop using browser based tool sets if it hasn't been minified. 

That being said I'd be happy to make the addition if you would still prefer.

Reply:
I do think running eslint on the build matters. It's good to check for errors at compile time. I'm fine with not minifying the code but I think some things may not work as they are expected to be minified versions like the js files. So, if the plan is to not run minified we need changes to the HTML pages as well.

Reply:
Whether the JS is minified or not should have no effect on the resulting behavior. If you can provide me something that says otherwise I'd love to read it. HTML behavior differs when it's minified, as such it should never be minified.

As for the running ESLint, that touches a different subject where my opinion again differs. 😄 Again this is a development docker image that should never be published to an image registry. Therefore it shouldn't matter. That being said, I would suggest setting up something like "Husky" (It's an NPM package) that will lint your JS on commits. 

Again, something else you could do to simplify handling of building images and pushing them is to setup a "GitHub Action" to build images and push them to the registry when you tag a release in GitHub. Then you'll no longer have to do that by hand and that can all be automated moving forward. If you want I would happily give you an example.

P.S. I wish there was a more "real-time" way to discuss these things baked into GitHub. 😄

Reply:
It's assuming the minified plugins aren't broken or misconfigured. By testing against minified as a pre-production release, I can better verify everything is good for a release. It's fine to leave out. I can run it manually.

I can't stand GitHub actions. I had them set up on this repo and removed them. I run a Jenkins docker to do builds and unit testing. I like the idea of Husky but ESLint confirms code is built well and not in violation. It also catches dumb errors for developers saving build time. I'm fine with no minified on dev but ESLint I would want.

Reply:
Pushed.